bigflow/cli.py

```diff
@@ -812,5 +831,7 @@ def cli(raw_args) -> None:
         cli_logs(root_package)
     elif operation == 'build-requirements':
         _cli_build_requirements(parsed_args)
+    elif operation == 'codegen':
```

Comment:
Maybe something more problem-related, like `bf fix-dataflow-dependency-conflicts`

Reply:
I don't very like to "litter" `bigflow` cmd-line interface with such "very occasional and specific" functions. One indicator  why it is "specific" case - very long and "ugly" name (either current `bf codegen pin-dataflow-requirements` or proposed `bf fix-dataflow-dependency-conflicts`).

Idea was to add a new "umbrella" command for such tools - eventually we might want to add something like `upgrade-docker-file-with-new-python`, `create-pyproject-toml`, `pin-dataproc-requirements`, `migrate-to-bigflow-1.2` etc etc ...)

I'm not insist on this idea - we can just put all such things into a global namespace if you would prefer.

Regarding actual command name (`pin-dataflow-requirements` vs `fix-dataflow-dependency-conflicts`).
IMO this is not very important, as this command is printed to user as a warning "please run this xxx to fix an issue".

Reply:
Idea was to add a new "umbrella" command for such tools - eventually we might want to add something like upgrade-docker-file-with-new-python, create-pyproject-toml, pin-dataproc-requirements, migrate-to-bigflow-1.2 etc etc ...)

ok